components/Footer.js

```diff
@@ -55,6 +55,14 @@ export default function Footer (props) {
                 API documentation
               </NavLink>
             </li>
+            <li>
+              <NavLink
+                href={process.env.OTP_DEVGROUP_URL}
+                external
+              >
+                OpenTripPlanner Developers Google Group
```

Comment:
I think we should call this "OTP Developers Forum". It fits the container a little better and the project requirement requires us to provide a "developers forum", so I think it matches that requirement a little better also.

Reply:
Updated in https://github.com/ibi-group/otp-admin-ui/pull/28/commits/465cffdbf274a02b25f2c7500a568c2daf349046.